### DISCUSSION TITLE
What's the ConnectAPI equivalent of ProvisioningProfile.update ?

### DISCUSSION THREAD

**Discussion by who** [fastlane/fastlane#18060] [Q&A]
I see that `Spaceship::Portal::ProvisioningProfile` has an `update!` method.

However, in `Spaceship::ConnectAPI::Profile`, I don't see `update!`.  Should I be calling `Spaceship::ConnectAPI::Profile.create` for updating purposes?

**Comment by dinsen** [comment]
Hi @who 
There is no update endpoint for the App Store Connect API, so you will have to use the method you already mention.
